src/pages/contributing/product-development-lifecycle/index.mdx

```diff
@@ -0,0 +1,145 @@
+- Identify 5–8 stakeholder teams for early usage and feedback
+- Backlog work begins on kit, docs, code triumvirate per definition of done
+- Any breaking changes are integrated into the Carbon library behind a [feature flag](/components/overview/feature-flags/)
+
+#### Prioritization
+
+Once a contribution enters the delivery phase it must be prioritized against the other contributions and work streams. The most important factor in determining prioritization in the contribution pipeline is business impact. The greater the case for reuse or support for a high impact team, the more likely a proposal is to move up in the pipeline and garner more Carbon resources.
+
+#### Delivery status
+
+All assets in the delivery phase are considered in `preview`. Carbon formerly called this status “experimental” but we wanted to increase the confidence and requirements of this status to be available for use in production. However, it is not stable yet and breaking changes are still possible based on feedback from use in products. 
+
+## Launch and scale 
+
+Launch and scale of course, involves having clearly defined success criteria or learning objectives that are monitored after launch. In this phase an asset will become stable and achieve Carbon’s “definition of done” as defined below.
+
+### Launch and scale criteria
+
+In the delivery phase, the workgroups should begin to think about the requirements for a component or pattern to become “stable.” In the launch and scale phase, all of those requirements must be met. 
+
+#### Full, peer-review completed requirements
+Along the way you should be requesting peer on the various deliverables. It is crucial to get reviews early and often to make sure all requirements are accounted for. Reach out to the Carbon team if you are unsure who should review your work.
+
+#### Launch and scale status
+All components in the in launch and scale phase are `stable`. This means all requirements in the assets [definition of done](/contributing/product-development-lifecycle/#definition-of-done) are complete and the asset is ready to use in production.
+
+#### Final steps
+- Once an asset is complete there should be a communication plan in place to raise awareness of the new work across multiple channels. 
+- PMs should also begin to track the usage (product insertions) of the new asset via Figma’s API and the IBM Telemetry service.
+
+## Definition of done
+
+By aligning on the requirements for what it means for an asset to be done, we can create a backlog of work to be prioritized, better differentiate when an asset is a component versus a pattern, share expectations with contributors, and display the status of assets to users. With each phase, the component should progress in its completeness. Once it has reached stable, then the asset will be considered done.
+
+| Status              | PDLC Phase       | Description                                                                          |
```

Comment:
I think Phase in this column header should be lower case, but will leave that up to you to decide.